Library/Homebrew/formula.rb

```diff
@@ -143,9 +148,10 @@ class Formula
   attr_accessor :build
 
   # @private
-  def initialize(name, path, spec)
+  def initialize(name, path, spec, install_name: name)
```

Comment:
We're still mostly using hash rockets until we mass change them at some later point. Were still on Ruby 1.8 until recently!